src/apispec/core.py

```diff
@@ -360,14 +265,108 @@ def path(
             except PluginMethodNotImplementedError:
                 continue
 
-        clean_operations(operations, self.openapi_version.major)
+        self.clean_operations(operations, self.openapi_version.major)
 
         self._paths.setdefault(path, operations).update(operations)
         if summary is not None:
             self._paths[path]["summary"] = summary
         if description is not None:
             self._paths[path]["description"] = description
         if parameters:
-            parameters = clean_parameters(parameters, self.openapi_version.major)
+            parameters = self.clean_parameters(parameters, self.openapi_version.major)
             self._paths[path]["parameters"] = parameters
         return self
+
+    @staticmethod
+    def get_ref(obj_type, obj, openapi_major_version):
```

Comment:
Can you make this an instance method and get ride of the `openapi_major_version` arg?

Reply:
Good catch. Done.